core/src/test/resources/test_clients.yaml

```diff
@@ -1,12 +1,14 @@
-custodial:
+items:
```

Comment:
I though it would look a little bit redundant when there are two clients nested together:
```
clients:
  type:
  clients:
```
I had a discussion with @lijamie98 that we might want use `items` for all the list configuration eventually.